### DISCUSSION TITLE
Open source contractors wanted

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#7930] [Announcements]
> [!NOTE]
> Although this discussion is closed, we're still looking to fund contributors for their efforts on Deskflow. If you're interested in working on Deskflow and getting paid for your efforts, please do let us know.
> 
> [![Good first issues](https://img.shields.io/github/issues/deskflow/deskflow/good%20first%20issue?label=good%20first%20issues&color=%2344cc11)](https://github.com/deskflow/deskflow/labels/good%20first%20issue) [![Open bounty issues](https://img.shields.io/github/issues/deskflow/deskflow/%F0%9F%92%8E%20bounty?label=💎%20open%20bounty%20issues&color=%2344cc11)](https://github.com/deskflow/deskflow/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22%F0%9F%92%8E%20bounty%22) [![Rewarded bounties](https://img.shields.io/github/issues-search/deskflow/deskflow?query=label%3A%22%F0%9F%92%B0%20rewarded%22&label=%F0%9F%92%B0%20rewarded%20bounties&color=yellow)](https://github.com/deskflow/deskflow/issues?q=label%3A%22%F0%9F%92%B0%20rewarded%22%20sort%3Aupdated-desc)

[![Sponsored by Synergy](https://raw.githubusercontent.com/deskflow/deskflow-artwork/b2c72a3e60a42dee793bd47efc275b5ee0bdaa5f/misc/synergy-sponsor.svg)](https://symless.com/synergy)

The Synergy/Deskflow team is currently gathering quotes from contractors to:

1. Add the missing [clipboard sharing support](https://github.com/flatpak/xdg-desktop-portal/discussions/1459) to `flatpak/xdg-desktop-portal` (and/or whatever else is missing to enable Wayland clipboard access).
2. Add/finish the missing [portal permission dialog persistence](https://github.com/flatpak/xdg-desktop-portal/pull/1186) feature to `flatpak/xdg-desktop-portal`.
3. Port [CVE TLS fixes](https://github.com/deskflow/deskflow/issues/7806) from Input Leap to Deskflow.

The XDG Desktop Portal features will be available to implement in all Synergy-like apps (Deskflow, Input Leap, and Lan Mouse).

If you know of any open source contractors with experience working on the OS itself, please reply here or contact me on Matrix: https://matrix.to/#/#deskflow:matrix.org

**Comment by sithlord48** [comment]
With all the tasks here bein completeted i think you can close this discussion.
